### PR TITLE
Added html templates

### PR DESCRIPTION
**Pull request by atmiguel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Assuming your local environment is setup correctly with Golang standards, you ca
 
 `cd to this repo`
 
+`export PORT=8080`
```
Comment (zingales) [inline README.md:7]: So Heorku uses environment variables for configurations. And it also has a deploy local command, that will populate certain environment fields from a json file. I'll look into it but once i've set that up we will no longer need to do this manually.

File: main.go [hunk 2]
```diff
@@ -10,25 +11,21 @@ import (
...
 
-    var responseContent string = fmt.Sprintf(
-        "Hi there, I love %s!",
-        request.URL.Path[1:])
-
-    fmt.Fprintf(
-        responseWriter,
-        responseContent)
+    // TODO handle error
+    parsedTemplate, _ := template.ParseFiles("root.tmpl")
```
Comment (zingales) [inline main.go]: I see you have the todo, but can we default to just log errors? I just don't like silent bugs, i'd rather have to sift through too many logs than have to find an error that is being swallowed

File: root.tmpl [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
```
Comment (zingales) [inline root.tmpl:1]: Lets make a separate folder for templates.

**Review by zingales — CHANGES_REQUESTED** [review]

File: root.tmpl (hunk 3 above)
Comment (atmiguel) [reply root.tmpl:1]: done in a future branch

File: main.go (hunk 2 above)
Comment (atmiguel) [reply main.go]: will do. though how do we decide when to `log.Fatal()` vs `panic()`?

**Review by zingales — APPROVED** [review]

File: main.go (hunk 2 above)
Comment (zingales) [reply main.go]: I think panic should be avoided unless the program literally cannot continue. 

https://www.reddit.com/r/golang/comments/2a1sie/when_to_call_panic/ciqn81u/

Comment (atmiguel) [reply main.go]: okay we've gotta figure out a succinct way of returning a 500 in these situations

**Pull request merged**